<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_2DWPUClass</class>
 <widget class="QMainWindow" name="GUI_2DWPUClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI_2DWPU</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="programBlockInstrTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>8</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>32</number>
    </property>
    <property name="columnCount">
     <number>32</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>15</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string>Test</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="BsimStep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="coresInfoT">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>256</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="BsimStop">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BsimMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="frequencyL">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="numDigits">
     <number>12</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionLoad_HEX"/>
    <addaction name="actionOpen_2DASM"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionSet_Cores"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBasic_Instruction_Set_Analyzer"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBasic_Instruction_Set"/>
   </widget>
   <widget class="QMenu" name="menuIO_interface">
    <property name="title">
     <string>IO interface</string>
    </property>
    <addaction name="actionSimpleLCD_400x240"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuIO_interface"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBasic_Instruction_Set_Analyzer">
   <property name="text">
    <string>Basic Instruction Set Analyzer</string>
   </property>
  </action>
  <action name="actionBasic_Instruction_Set">
   <property name="text">
    <string>Basic Instruction Set</string>
   </property>
  </action>
  <action name="actionLoad_HEX">
   <property name="text">
    <string>Open HEX</string>
   </property>
  </action>
  <action name="actionOpen_2DASM">
   <property name="text">
    <string>Open 2DASM</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>New (clear RAM)</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionSet_Cores">
   <property name="text">
    <string>Set Cores</string>
   </property>
  </action>
  <action name="actionSimpleLCD_400x240">
   <property name="text">
    <string>SimpleLCD (400x240)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui_2dwpu.qrc"/>
 </resources>
 <connections/>
</ui>
